src/StaticFAQ/FAQItem.js

```diff
@@ -31,34 +29,48 @@ const styles = css`
 `;
 
 type Props = {|
-  category: FAQCategory_category,
+  data: FAQItemCategory | FAQItemArticle,
 |};
 
-const FAQCategory = (props: Props) => (
-  <div className="faq-category">
+// F.I.X.M.E. - perex for categories needs to be added to the API
+const defaultCategoryPerex = 'How to find your best deal fast and easy';
+
+const FAQItem = (props: Props) => (
+  <div className="faq-item">
     <div>
       <Typography type="attention" size="large">
-        {props.category.title}
+        {props.data.title}
       </Typography>
     </div>
     <div>
       <Typography type="secondary" size="small">
-        How to find your best deal fast and easy
+        {props.data.perex || defaultCategoryPerex}
       </Typography>
     </div>
-    <div className="faq-category-chevron">
+    <div className="faq-item-chevron">
       <ChevronRight height="12" />
     </div>
     <style jsx>{styles}</style>
   </div>
 );
 
-export default createFragmentContainer(
-  FAQCategory,
+export const FAQCategory = createFragmentContainer(
+  FAQItem,
+  graphql`
+    fragment FAQItemCategory on FAQCategory {
+      id
+      title
+    }
+  `,
+);
+
+export const FAQArticle = createFragmentContainer(
+  FAQItem,
```

Comment:
Just notice that article and categories have a slightly different design, I'll be creating a common article card/box component, so for now is ok.

Reply:
Ha! ok, I see, then I will refactor that into two independent components